### ISSUE TITLE
Menu doesn't announce collapsed or expanded on mobile

### ISSUE THREAD

**Issue by Lu-stack-art** [cds-snc/design-gc-conception#1592]
**Mobile device, iOS, using VoiceOver, on Safari**

- "Menu at the top does not indicate expand or collapse, would be nice to have that feature included for easier navigation/notification for the user (especially on mobile)"
- https://cds-snc.dovetail.com/data/Fable-1st-User-Interview-Moderated-17TO4TOEEcFhe2mUZyG9Jz
- https://app.makeitfable.com/requests/CDS-CGDS-48 - 41:30
